src/Sarif.Driver/FileHelpers.cs

```diff
@@ -15,9 +13,9 @@ namespace Microsoft.CodeAnalysis.Sarif.Driver
     /// </summary>
     static class FileHelpers
     {
-        public static T ReadSarifFile<T>(string filePath, IContractResolver contractResolver = null)
+        public static T ReadSarifFile<T>(IFileSystem fileSystem, string filePath, IContractResolver contractResolver = null)
```

Comment:
>fileSystem [](start = 53, length = 10)

Thank you!

In a future PR (or this one, if you're feeling ambitious), take a look at making `FileHelpers` and instance class, passing the `IFileSystem` in the ctor, and then you wouldn't have to pass it on each call.